test/test_sparql_converter.js

```diff
@@ -137,14 +138,14 @@ const TEST_CASES = [
     join ([id] of @org.wikidatasportsskill.sports_team())), P647 == lhs.id && P286 == rhs.id => notify;
         `,
         `
-    SELECT distinct ?v1Label WHERE{
```

Comment:
Why are other examples above not affected? Shouldn't they all be using id instead of v1?